R/zed-scores.R

```diff
@@ -0,0 +1,90 @@
+# z-score calculations from Doug's tables pdf. This will soon be deprecated and
+# replaced using the cross product of the margins for expected values.
+zScoresDep <- function (table, margin) {
+    other_margin <- 3 - margin ## Assumes 2-D
```

Comment:
What happens if you give this function a 3D cube? or a cube with an array?

Reply:
They both work out of the box now (with tests!)